src/components/themed/ExchangedFlipInput2.tsx

```diff
@@ -8,11 +8,13 @@ import { useDisplayDenom } from '../../hooks/useDisplayDenom'
 import { useHandler } from '../../hooks/useHandler'
 import { lstrings } from '../../locales/strings'
 import { getExchangeDenom } from '../../selectors/DenominationSelectors'
-import { getExchangeRate } from '../../selectors/WalletSelectors'
+import {
+  convertCurrency as convertCurrencyFromExchangeRates,
```

Comment:
Why rename the import? Was this a mistake?

Reply:
"convertCurrency" was declared elsewhere in the file. I'll change that name instead